test/quantization/fx/test_quantize_fx.py

```diff
@@ -1033,6 +1033,67 @@ def forward(self, x):
         self.assertEqual(m.module_conv1.qconfig, module_name_regex_qconfig)
         self.assertEqual(m.module_conv2.qconfig, module_name_qconfig)
 
+    def test_qconfig_module_name_function_order(self):
+        class M1(torch.nn.Module):
+            def forward(self, x):
+                x = torch.add(x, x)
+                x = torch.add(x, x)
+                return x
+
+        class M2(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.m1 = M1()
+
+            def forward(self, x):
+                x = torch.add(x, x)
+                x = torch.add(x, x)
+                x = self.m1(x)
+                return x
+
+        class M3(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.m2 = M2()
+
+            def forward(self, x):
+                x = torch.add(x, x)
+                x = torch.add(x, x)
+                x = self.m2(x)
+                return x
+
+        m = M3().eval()
+        qconfig_dict = {
```

Comment:
A common use case is that users want to specify a few layers to skip quantization in qconfig_dict, while most other layers are kept quantized using some global qconfig, in which case it's more convenient to specify layers to skip quantization in qconfig_dict instead of layers to quantize. How could users setup qconfig_dict for such use case?

Reply:
one could specify

```
qconfig_dict = {
  "module_name_function_order": ("m1", torch.add, 0, None),
}
```

to skip quantization of the first add, and so forth